docs/service-bundle.md

```diff
@@ -127,12 +129,18 @@ Example of JSON document passed to __bind__:
   provision action for this service instance. Details are below in the Output
   section.
 
+#### Last Request User
+
+The requesting username of the [actions](#actions) _provision_, _deprovision_, _bind_, _unbind_, and _update_ will available in the `_apb_last_requesting_user` parameter. The parameter value will be the `UID` of the requesting user if the `username` is not available (e.g. auto escalation). Also, the user info of the latest request will always overwrite the existing value.
+
+This feature will be available in v3.10
```

Comment:
I think this should be a broker version, right?

Reply:
As a nit-pick, I think it makes sense for these to be present-tense. "This field is new in version X.Y." Or even past tense: "This field was introduced in version X.Y."

Users aren't likely to care about this field or read about it until after its release.

Reply:
`release-1.2`?

Reply:
oh, ok, so I'll make those updates

Reply:
I think I'd go with just the version number rather than a branch name, but either way I think that's the best number to convey.